core/src/rpc.rs

```diff
@@ -2322,6 +2322,10 @@ impl RpcSol for RpcSolImpl {
             Ok(())
         };
 
+        if let Err(e) = transaction.verify_precompiles() {
```

Comment:
I think this should be under the `if config.sig_verify` condition above

Reply:
Roger!